hazelcast/src/main/java/com/hazelcast/client/impl/protocol/task/management/GetTimedMemberStateMessageTask.java

```diff
@@ -22,11 +22,14 @@
 import com.hazelcast.instance.impl.Node;
 import com.hazelcast.internal.management.ManagementCenterService;
 import com.hazelcast.internal.nio.Connection;
+import com.hazelcast.security.permission.ManagementPermission;
 
 import java.security.Permission;
 
 public class GetTimedMemberStateMessageTask extends AbstractCallableMessageTask<Void> {
 
+    public static final Permission REQUIRED_PERMISSION = new ManagementPermission("getTimedMemberState");
```

Comment:
Wouldn't it be better if we grouped the MC tasks under a `mc.*` group?

Reply:
What do you mean by MC tasks? All of the tasks are used in MC and none of them are restricted to MC (i.e. they can be used by anyone that implements the client side of these operations). 

I would group this under `member` and call it `member.getTimedMemberState` since it's specific to one member but it's not super important.

Reply:
I meant the ones with no prefix, but you're right.

Reply:
:+1: